meals/templates/include/meals_tab.html

```diff
@@ -0,0 +1,3 @@
+{% if request.user.is_organizer %}
+    <li><a href="{% url 'meals_list' %}">Meals</a></li>
```

Comment:
`...<a class="{% if 'meals' in request.build_absolute_uri %}active{% endif %}">...`

Reply:
Also noted in #52, ty.

Reply:
Added.